src/bindings/html/langs.js

```diff
@@ -68,11 +17,7 @@ export function negotiateLanguages(
     src: getLangSource(appVersion, availableLangs, additionalLangs, code),
   }));
 
-  if (!arrEqual(prevLangs, newLangs)) {
-    fn(langs);
-  }
-
-  return langs;
+  return [langs, !arrEqual(prevLangs, newLangs)];
```

Comment:
since you're passing this around, it would be more readable if it was an object rather than an array.